lib/etna/components/errors/json_parse_error.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Etna
+  module Components
+    module Errors
+      class JsonParseError < Base
+
+        attr_reader :message, :exception, :backtrace
+
+        def initialize(typhoeus_response, request, exception, message = nil, caller_backtrace: caller)
+          @response = typhoeus_response
+          @request = request
+
+          @status = 499
```

Comment:
I don't know if `499` is the right status here, as far as[ I know that code is nginx specific](https://httpstatuses.com/499).

Wouldn't `400` be better because whatever payload we sent could not be handled properly by the receiver?